audit/projects/k8s-conform/buckets/k8s-conform-cri-o/iam.json

```diff
@@ -23,15 +23,10 @@
     {
       "members": [
         "group:k8s-infra-artifact-admins@kubernetes.io",
-        "group:k8s-infra-conform-cri-o@kubernetes.io"
-      ],
-      "role": "roles/storage.objectAdmin"
-    },
-    {
-      "members": [
+        "group:k8s-infra-conform-cri-o@kubernetes.io",
         "serviceAccount:service-cri-o@k8s-conform.iam.gserviceaccount.com"
       ],
-      "role": "roles/storage.objectCreator"
+      "role": "roles/storage.objectAdmin"
```

Comment:
???

did someone make this change manually?

Reply:
weird! (not me!) 

cc @saschagrunert @mrunalp any clues?

Reply:
Aww, yes I changed it manually. Sorry for that but our token needs access to write a version marker to the bucket. :-/ We need to change that file for each commit.

Reply:
Can I ask what the version marker is for? These buckets should just be result dumps, just trying to understand the use case here.

OTOH since we're giving humans admin access I can't think why we wouldn't give their serviceaccount the same level of acess.  I would be open to a PR that makes this the default for all k8s-conform buckets, WDYT @BenTheElder ?

Reply:
/cc @BenTheElder 
to put the above question on your radar

Reply:
> Can I ask what the version marker is for? These buckets should just be result dumps, just trying to understand the use case here.

Yes sure, the main intention was to use this marker for being independent from the GitHub API. We publish a binary artifact for every successful run on the CRI-O master branch and update the version marker after that. This way we can easily query the latest build without having to use the rate limited GitHub Actions API. It's more or less the same like we do it in k8s.

Reply:
Opened https://github.com/kubernetes/k8s.io/issues/1850 to track following up on this

Reply:
FYI I mistakenly deleted this account when trying to delete the capi-openstack serviceaccount below.  I believe I restored it, but let me know if you see problems on your end @saschagrunert 
```
$ gcloud iam service-accounts delete service-cri-o@k8s-conform.iam.gserviceaccount.com
deleted service account [service-cri-o@k8s-conform.iam.gserviceaccount.com]
$ gcloud beta iam service-accounts undelete 118310596454734433596
restoredAccount:
  email: service-cri-o@k8s-conform.iam.gserviceaccount.com
  etag: MDEwMjE5MjA=
  name: projects/k8s-conform/serviceAccounts/service-cri-o@k8s-conform.iam.gserviceaccount.com
  oauth2ClientId: REDACTED
  projectId: k8s-conform
  uniqueId: '118310596454734433596'
```